octobot_backtesting/api/exchange_data_collector.py

```diff
@@ -27,6 +27,24 @@ async def collect_exchange_historical_data(exchange_name,
     collector_instance = collector_class({}, exchange_name, tentacles_setup_config, symbols, time_frames,
                                          use_all_available_timeframes=time_frames is None,
                                          start_timestamp=start_timestamp, end_timestamp=end_timestamp)
-    await collector_instance.initialize()
-    await collector_instance.start()
-    return collector_instance.file_name
+    return collector_instance
+
+async def initialize_and_run_data_collector(data_collector):
+    await data_collector.initialize()
+    await data_collector.start()
+    return data_collector.file_name
+
+def is_data_collector_in_progress(data_collector):
+    if data_collector:
+        return data_collector.is_in_progress()
+
+def get_data_collector_progress(data_collector):
+    if data_collector:
+        current_time_frame_index = data_collector.get_current_time_frame_index()
+        current_time_frame_percent = data_collector.get_current_time_frame_percent()
+        total_time_frame = data_collector.get_total_time_frame()
```

Comment:
Can you just return the method call result instead of creating local variables please?